introduction/introduction.md

```diff
@@ -65,5 +65,15 @@ This wiki is the main source of documentation for the Pupil users getting starte
 				</div>
 			</div>
 		</a>
+		<a href="https://discordapp.com/invite/gKmmGqy">
```

Comment:
link can be our redirect to permanent invite: https://pupil-labs.com/chat

Reply:
external link should be rel=noopener -- see info on this method [here](https://mathiasbynens.github.io/rel-noopener/)